src/DocScan/Session/Create/Traits/Builder/ManualCheckTrait.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yoti\DocScan\Session\Create\Traits\Builder;
+
+use Yoti\DocScan\Constants;
+
+trait ManualCheckTrait
```

Comment:
This could be moved into a `BaseTextExtractionTaskBuilder` that text extraction task builders extend if that would be clearer? (this would involve reinstating these duplicate methods in `RequestedFaceMatchCheckBuilder`)

Reply:
I think this looks clear as is 👍